packages/doc/src/components/Home/Box.jsx

```diff
@@ -0,0 +1,110 @@
+import React from 'react';
+import { number, string } from 'prop-types';
+import styled from 'styled-components';
+
+const Item = styled.a(
+  ({
+    theme: {
+      yoga: {
+        colors: { gray, dark },
+      },
+    },
+  }) => `
+  height: 284px;
+  margin-right: 24px;
+  padding: 24px;
+
+  display: flex;
+  flex-basis: 300px;
+  flex-direction: column;
+  flex-grow: 1;
+  justify-content: space-between;
+
+  background-color: ${gray[1]};
+  color: ${dark};
+  text-decoration: none;
+
+  &:last-child {
+    margin-right: 0;
+    margin-bottom: 0;
+  }
+
+  @media (max-width: 900px) {
+    flex-basis: 100%;
+    margin: 9px 0;
+
+    height: auto;
+  }
+`,
```

Comment:
I think that we can adjust this component following the CSS Ordering, what do you think?

Reply:
Done!